testsuite/mockserver.py

```diff
@@ -40,3 +73,12 @@ def clear_expectation(self, expectation_id):
                     "id": expectation_id
                 }
             ).raise_for_status()
+        del self.expectations[expectation_id]
+
+    def verify_expectation(self, path):
```

Comment:
If you need just verification you can use https://www.mock-server.com/mock_server/verification.html which doesn't require a expectation at all (see https://github.com/3scale-qe/3scale-tests/pull/488)

Reply:
The verification endpoint does not provide information about the number of requests received.

Reply:
And I need expectation for random UUID